styles/style.css

```diff
@@ -17,6 +17,11 @@ button {
 
 /* Navigation */
 
+.donate_now {
```

Comment:
Don't forget to add the common styles to all buttons further down where it says /*INSERT BUTTON STYLES HERE*/